src/content/release-notes.mdx

```diff
@@ -5,6 +5,17 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.20.1
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### Improvements
+
+- Updated CLI version in the backend to 2.27.3  <!-- 8135 -->
```

Comment:
Let's elaborate a bit more this indicating why. We could set something like:

`Updated CLI version in the backend to 2.27.3 to fix a panic when using Docker Compose and Okteto Manifests together, and the field image in a compose service was referring to an environment variable`

@codyjlandstrom Any better suggestion?

Reply:
I've updated with this suggestion, feel free to update it while I am away